concourse/steps/update_component_deps.py

```diff
@@ -398,8 +399,16 @@ def create_upgrade_pr(
 
     if merge_policy is MergePolicy.MANUAL:
         return
-    # auto-merge - todo: make configurable (e.g. merge method)
-    pull_request.merge()
+
+    if merge_method is MergeMethod.MERGE:
```

Comment:
just use `merge_method.value`? checking for unknown / unimplemented values is fine, of course